Anytype/Sources/Analytics/Converters/FileDetails+Analytics.swift

```diff
@@ -0,0 +1,7 @@
+import Services
+
+extension FileDetails {
+    var analyticsType: AnalyticsObjectType {
+        .file(fileExt: fileExt)
```

Comment:
Maybe add lowercase conversion? Extension can be: JPEG, Jpeg, jpeg...

Reply:
totally agree, will add!